### PR TITLE
Update splice.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Closing [this PR](https://github.com/enkidevs/curriculum/pull/2492) because of the structural changes and creating this one with the same changes.

Changes:
- Add missing information regarding the *delete* argument for the Splice function

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/javascript-core/javascript-arrays/splice.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/javascript-core/javascript-arrays/splice.md?ref=26c0dae1e4fe46a16157b4aade51a48c0e5da6d4)

File: javascript/javascript-core/javascript-arrays/splice.md [hunk 1]
```diff
@@ -18,11 +18,13 @@ links:
 
 Splice is a handy array function that allows you to add and remove items to an array.
 
-It accepts 3 arguments with the 3rd being optional:
+It accepts 3 arguments with the 2nd and 3rd being optional:
```
Comment (kapnobatai137) [inline javascript/javascript-core/javascript-arrays/splice.md:21]: how does this sound?

```suggestion
It accepts three arguments, with **only the first being mandatory**:
```

File: javascript/javascript-core/javascript-arrays/splice.md [hunk 2]
```diff
@@ -18,11 +18,13 @@ links:
...
 
 - *Start* is the 0 based index of where to start manipulating the array
 - *Delete* is how many items to remove at the *Start* position (can be 0)
-- *Items to add* (optional), separated by commas
+- *Items to add*, separated by commas
+
+If *Delete* is omitted, splice will remove all elements from the specified *Start* to the end of the array. On the other hand, if its 0 or a negative number, it won't delete any elements.
```
Comment (kapnobatai137) [inline javascript/javascript-core/javascript-arrays/splice.md:27]:
```suggestion
If the *Delete* argument is omitted, `splice` will remove all elements from the specified *Start* to the end of the array. On the other hand, if it's `0` or a negative number, it won't delete any elements.
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**